### PR TITLE
Updated Discord Server Invite Link

### PR DESCRIPTION
**Pull request by kailashchoudhary11**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug Fix
- **Why was this change needed?** (You can also link to an open issue here)
Since the previous link has expired, members were not able to join the discord server!
- **Other information**:

### CODE REVIEW

**Comment by kailashchoudhary11** [comment]
@davidsoderberg @nevo-david Please Review!

**Review by BoseSj — COMMENTED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ To ensure consistency throughout the source code, please keep these rules in min
 
 ## Need help? Questions and suggestions
 
-Questions, suggestions and thoughts are most welcome. Feel free to open a [Github Discussion](https://github.com/novuhq/novu/discussions). We can also be reached in our [Discord Channel](https://discord.gg/heTZ9zJd).
+Questions, suggestions and thoughts are most welcome. Feel free to open a [Github Discussion](https://github.com/novuhq/novu/discussions). We can also be reached in our [Discord Channel](https://discord.gg/R5HDTka5).
```
Comment (jainpawan21) [inline CONTRIBUTING.md:56]: Please use this discord link https://discord.gg/novu

**Review by jainpawan21 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kailashchoudhary11) [reply CONTRIBUTING.md:56]: Okay Sure!

**Review by jainpawan21 — APPROVED** [review]
Thanks 🙏🏻

**Comment by kailashchoudhary11** [comment]
@davidsoderberg Please Review!

**Review by davidsoderberg — APPROVED** [review]

**Pull request merged**